app/views/availabilities/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="container-fluid">
+  <h1>Schedule Availability</h1>
+  <%= link_to 'Logout', destroy_user_session_path, method: :delete %>
+  <%= form_tag("/availabilities", method: "post") do %>
+    <table class="table table-sm">
+      <tr>
+        <th></th>
+        <% start_time = @start_time %>
+        <% end_time = @end_time %>
+        <% while start_time <= end_time do %>
+          <th colspan="2">
+            <span class="time-label">
+              <%= start_time.strftime("%H:%M") %>
+            </span>
+          </th>
+          <% start_time += 1.hour %>
+        <% end %>
+      </tr>
+      <% @availabilities.each do |availability_day| %>
+        <tr>
+          <td class="dayname"><%= availability_day[0].day[0..2] %></td>
+          <% availability_day.each do |availability| %>
+            <td align="center" class="checkbox_cell"
```

Comment:
This massive tag, put in helper. And use http://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-tag

Reply:
For the `availability_day[0]`, need to take the day of current day, where all availabilities in the availability_day array have the same day.

Reply:
For the `availability_day[0]`, need to take the day of current day, where all availabilities in the availability_day array have the same day.